CStdString.h

```diff
@@ -690,6 +690,48 @@ inline const Type& SSMAX(const Type& arg1, const Type& arg2)
 	#include <locale>			// for various facets
 #endif
 
+// strspn, strcspn
+#ifdef SS_WIN32
+	#ifdef SS_ANSI
+		#include <string.h>
+		#ifdef _UNICODE
+			#define _tcsspn wcsspn
+			#define _tcscspn wcscspn
+		//#elif defined(SS_MBCS)
```

Comment:
tbh I don't know a whole lot about MBCS since we've never used it.  Is this a case we have to consider?